### ISSUE TITLE
Gothic 1: flickering of some textures for some seconds after hero stops walking

### ISSUE THREAD

**Issue by lucifer602288** [SaiyansKing/GD3D11#69]
Gothic 1: flickering of some textures for some seconds after hero stops walking.
Remove the texture update after the hero stopped walking.

Gothic 2 is fine.

**Comment by SaiyansKing** [comment]
Hi, do you use any "texturepack" because it seems that you describe what is called "z-fighting" and it cannot be erased completely on software level because floating point variables fixed-precision. I don't experience any texture-flickering on benches in original Gothic 1 game.

**Comment by lucifer602288** [comment]
Yes, I use RiisisGothic1TextureMixV1.1.vdf. Thank you for the explanation to understand it.
